.github/workflows/ci.yaml

```diff
@@ -0,0 +1,62 @@
+name: CI
+on:
+  - push
+  - pull_request
+
+jobs:
+  Quality:
+    strategy:
+      # Don't fast-fail on tag and master
```

Comment:
Can you explain (briefly) the needs behind this ?

Reply:
Might not be needed actually